pyiron/atomistics/structure/visualization.py

```diff
@@ -0,0 +1,624 @@
+# coding: utf-8
+# Copyright (c) Max-Planck-Institut für Eisenforschung GmbH - Computational Materials Design (CM) Department
+# Distributed under the terms of "New BSD License", see the LICENSE file.
+
+import numpy as np
+import warnings
+from matplotlib.colors import rgb2hex
+from pyiron_base import Settings
+from scipy.interpolate import interp1d
+
+__author__ = "Joerg Neugebauer, Sudarsan Surendralal"
+__copyright__ = (
+    "Copyright 2020, Max-Planck-Institut für Eisenforschung GmbH - "
+    "Computational Materials Design (CM) Department"
+)
+__version__ = "1.0"
+__maintainer__ = "Sudarsan Surendralal"
+__email__ = "surendralal@mpie.de"
+__status__ = "production"
+__date__ = "Sep 1, 2017"
+
+s = Settings()
+
+
+class Visualization(object):
```

Comment:
From the github diff it's not easy to see -- is this just all the old methods newly encapsulated as the Visualization object, or do some of the methods also have changed functionality? Actually, I'll take a quick peek at the commit messages to see if that clarifies things, but I might still want confirmation.

Reply:
Except for `ref_structure`, there shouldn't be any difference